spec/models/accounting/quickbooks/department_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
```

Comment:
I had a similar thought when I looked at the Department class, but I think this is fine. There is enough difference and not enough similarity to warrant use of inheritance at this point I think. But let's stay open to that possibility in the future.